### ISSUE TITLE
On the issue of supervision of the Loss Functions

### ISSUE THREAD

**Issue by xxl1995** [XunpengYi/Diff-IF#9]
作者你好，我想问一下损失函数的监督问题。我的理解如下：论文中使用了扩散损失、梯度损失和SSIM损失来训练网络，但是扩散模型的采样过程是不可微分的，因此扩散损失用于监督UNet，梯度损失和SSIM损失用于监督精细网络。请问我的理解是否正确？如果是这样的话，那如何保证扩散生成的样本是更优的、是任务所需的？ 精细化网络是不是比扩散网络更重要？
Dear Author,
I would like to inquire about the supervision of the Loss functions. My understanding is that the paper employs diffusion loss, gradient loss, and SSIM loss for training the network. However, the sampling process of the diffusion model is non-differentiable, so the diffusion loss is only used to supervise the UNet, while the gradient loss and SSIM loss are utilized to supervise the refinement network. May I ask if my understanding is accurate? 
If so, how can we ensure that the samples generated by diffusion are superior and suitable for the task?  Is the refinement network more crucial than the diffusion network?
Looking forward to your reply, thanks!

**Comment by XunpengYi** [comment]
扩散模型的监督来自于目标搜索的分布，这是在代理任务设定下的最优融合分布，因此扩散模型可以生成优秀的符合样本分布的融合结果。在细化网络中，一个重要损失是与最优融合分布对齐的像素损失，以保证扩散和细化结果的分布一致性，保持结果是尽可能最优的。其次才是你所说的梯度损失和SSIM损失，这是作为辅助优化融合结果。因此它们是一个整体，相互协作实现生成和约束，不存在哪个网络更加重要。
